module-check/src/test/clojure/clojure/core/typed/test/template.clj

```diff
@@ -0,0 +1,23 @@
+(ns clojure.core.typed.test.template
+  (:require 	[clojure.core.typed :as t]
+                [clojure.template :as template] 
+		[clojure.xml :as xml]
+		[clojure.zip :as zip]
+		[clojure.test :refer :all]                
+		[clojure.core.typed.test.test-utils :refer :all]))
+
+(deftest applyTemplate-test
+  (is-tc-e   (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5))t/Any) 
+  (is-tc-e   (clojure.template/apply-template '[a b c d e] {:a 1 :b 2 :c 3 :d 4 :e 5} '(1 2 3 4 5))t/Any)
+  (is-tc-e   (clojure.template/apply-template '[a b c d e] 3 '(1 2 3 4 5))t/Any)
+  (is-tc-err (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5)) (t/Vec t/Any))
+  (is-tc-err (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5)) (t/HVec t/Any))
+  (is-tc-err (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5)) (t/List t/Any)))
```

Comment:
These 3 `is-tc-err` doesn't make sense as they test exact same thing and only 1 is sufficient. Let's just remove second and third `is-tc-err` and see if Ambrose wants you to add more `is-tc-err` when he will be reviewing. Probably one is enough.